R/scholar.r

```diff
@@ -28,7 +28,7 @@ utils::globalVariables(c("name"))
 ##' @export
 ##' @importFrom stringr str_trim str_split
 ##' @importFrom xml2 read_html
-##' @importFrom rvest html_table html_nodes html_text
+##' @importFrom rvest html_table html_nodes html_text html_children
```

Comment:
Thanks @TS404 but you need to regenerate and commit the `NAMESPACE` file also for this import to be effective.